nautobot_floor_plan/svg.py

```diff
@@ -388,6 +479,62 @@ def _draw_rack_tile(self, drawing, tile):
                 style=f"fill: {fgcolor(tile.rack.status.color)}",
             )
         )
+        # Add a button for editing the tile definition
+        edit_url = reverse("plugins:nautobot_floor_plan:floorplantile_edit", kwargs={"pk": tile.pk})
+        query_params = urlencode({"return_url": self.return_url})
+        edit_url = f"{self.base_url}{edit_url}?{query_params}"
+        link = drawing.add(drawing.a(href=edit_url, target="_top"))
+        link.add(
+            drawing.rect(
+                (origin[0] + self.TILE_INSET + self.RACK_BUTON_OFFSET, origin[1] + self.TILE_INSET + self.GRID_OFFSET),
+                (self.TEXT_LINE_HEIGHT, self.TEXT_LINE_HEIGHT),
+                class_="edit-tile-button",
+                rx=self.CORNER_RADIUS,
+            )
+        )
+        link.add(
+            drawing.text(
+                "✎",
+                insert=(
+                    origin[0] + self.TILE_INSET + self.TEXT_LINE_HEIGHT / 2 + self.RACK_BUTON_OFFSET,
+                    origin[1] + self.TILE_INSET + self.TEXT_LINE_HEIGHT / 2 + self.GRID_OFFSET,
+                ),
+                class_="button-text",
+            )
+        )
```

Comment:
Would it be possible/appropriate to create some reusable helper functions for the "edit" and "delete" button rendering? Feels like a lot of repeated code for these.

Reply:
This is now a edit and delete function and the function _draw_defined_status_tiles has been removed.